components/add_new_cv_form.tsx

```diff
@@ -65,43 +53,13 @@ export default function AddNewCvForm({ id }: Props) {
   const [loading, setLoading] = useState(true);
 
   useEffect(() => {
-    const fetches = [fetchSkills()];
     if (id) {
-      fetches.push(fetchCv(id));
+      fetchCv(id);
     } else {
-      fetches.push(setAvailablePositions());
+      setAvailablePositions();
     }
-    Promise.all(fetches).then(() => setLoading(false));
```

Comment:
Setting loading flag inside those functions works fine, but I find it easier to understand what's going on by setting the loading flag inside the `useEffect`. This is fine though.